deployment/embedded-spring-rest/pom.xml

```diff
@@ -10,9 +10,10 @@
 
   <properties>
     <version.camunda>7.16.0</version.camunda>
-    <version.spring>5.2.8.RELEASE</version.spring>
+    <version.spring>5.2.14.RELEASE</version.spring>
```

Comment:
Should we go for the latest Spring 5 (i.e., 5.3.10)?

Reply:
👍